### PR TITLE
docs(suspensive.org): code transition

### LINKED ISSUE #1367: [Feature]: Dynamic docs with code animation by codehike or snappify or animate-code
### Package Scope

etc

### Description

https://codehike.org/

https://snappify.com/editor

https://github.com/onurkerimov/animate-code

### Possible Solution

Like below gif or React code

231688650-7cd957a9-8edc-4db8-a5fe-61e1c2179d91

### etc.

_No response_

### PR DESCRIPTION
**Pull request by manudeli**
close #1367 

# Code transition for main page

chrome-capture-2024-11-24 (1)

## PR Checklist

- [x] I did below actions if need

1. I read the [Contributing Guide](https://github.com/toss/suspensive/blob/main/CONTRIBUTING.md)
2. I added documents and tests.

### CODE REVIEW

**Review by SEOKKAMONI — COMMENTED** [review]
Here’s my small suggestion: if possible, it would be great to mention on the main page how Suspensive solves compatibility issues that arose when upgrading React Query from v4 to v5. Addressing compatibility challenges highlights why Suspensive is a special and meaningful library.

**Comment by manudeli** [comment]
@SEOKKAMONI 
I appreciate the suggestion and agree that it may help understanding now, but react-query 5 won't be a big issue in browsers in 1-2 years, so I'm not going to highlight it on the main page.

File: docs/suspensive.org/src/pages/ko/index.mdx
Comment (gwansikk) [inline docs/suspensive.org/src/pages/ko/index.mdx:1]: This property needs to be updated! `description="React Suspense를 쉽게 사용하기 위한 패키지들을 제공합니다."`

Comment (manudeli) [reply docs/suspensive.org/src/pages/ko/index.mdx:1]: Oh, I updated it! Thanks for your eagle eyes 👀🦅

**Comment by manudeli** [comment]
> I love it 👍👍

Thanks! Let's improve this gradually after merging🚀

**Pull request merged**